docs/configuration/access-control.md

```diff
@@ -26,29 +64,24 @@ A rule defines two things:
 
 The criteria are:
 
-* domain: domain targeted by the request.
-* resources: list of patterns that the path should match (one is sufficient).
+* domain: domain or list of domains targeted by the request.
+* resources: pattern or list of patterns that the path should match.
 * subject: the user or group of users to define the policy for.
 * networks: the network addresses, ranges (CIDR notation) or groups from where the request originates.
+* methods: the http methods used in the request.
 
-A rule is matched when all criteria of the rule match.
+A rule is matched when all criteria of the rule match. Rules are evaluated in sequential order, and this is
+particularly **important** for bypass rules. Bypass rules should generally 
```

Comment:
```suggestion
particularly **important** for bypass rules. Bypass rules should generally appear near the top of the rules list.
```

I suspect this is probably what you were trying to say here?

Reply:
Yep! Haha.